### PR TITLE
update heartbeats page to reflect new tracking functionality in SC/SP

### PR DESCRIPTION
**Pull request by PhilBastian**
document updates for:

- https://github.com/Particular/ServicePulse/pull/2019

### CODE REVIEW

File: monitoring/heartbeats/in-servicepulse.md [hunk 1]
```diff
@@ -6,38 +6,62 @@ redirects:
...
+
+By default, each new endpoint instance discovered by ServicePulse is monitored for heartbeats. If the instance is not sending heartbeats (either because it is offline or because it does not have the heartbeats plugin installed) it will immediately be marked as unhealthy and an alert will show in the navigation and on the dashboard. If this is not desired, E.g. if the instance is intentionally offline for maintenance or redeployment, then the instance can be muted which will prevent alerts from being shown. The endpoint will still be marked as unhealthy, as a reminder to unmute once maintenance is complete.
+
+There are bulk operation buttons at the top of the screen to allow for muting or unmuting all instances of an endpoint at once.
+
+### Removing Endpoint Instances
+
+When an endpoint instance has been decommissioned, and should no longer be displayed in ServicePulse, it can be removed. 
+- For [Tracked Instances](#configuration-track-instances), click the `Delete` button on the instance row to remove it from ServicePulse
+- For [Untracked Instances](#configuration-do-not-track-instances) the endpoint will automatically be removed after 6 hours of inactivity.
```
Comment (johnsimons) [inline monitoring/heartbeats/in-servicepulse.md:64]:
```suggestion
- For [Untracked Instances](#configuration-do-not-track-instances), the endpoint instance will automatically be removed after 6 hours of inactivity.
```

**Review by johnsimons — APPROVED** [review]
LGTM.
Should we name the endpoints better for the screenshots?

File: monitoring/heartbeats/in-servicepulse.md (hunk 1 above)
Comment (PhilBastian) [reply monitoring/heartbeats/in-servicepulse.md:64]: as far as the user is aware, there _are_ no instances, so it's the endpoint being removed not the instance

Comment (johnsimons) [reply monitoring/heartbeats/in-servicepulse.md:64]: it is the instances that are being auto-pruned, not the logical endpoint

**Comment by PhilBastian** [comment]
> Should we name the endpoints better for the screenshots?
done

**Pull request merged**